samples/Microsoft.Azure.WebJobs.Extensions.OpenApi.FunctionApp.Interfaces/Microsoft.Azure.WebJobs.Extensions.OpenApi.FunctionApp.Interfaces.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+      <TargetFramework>netstandard2.0</TargetFramework>
+      <AssemblyName>Microsoft.Azure.WebJobs.Extensions.OpenApi.FunctionApp.Interfaces</AssemblyName>
+      <RootNamespace>Microsoft.Azure.WebJobs.Extensions.OpenApi.FunctionApp.Interfaces</RootNamespace>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <PackageReference Include="Microsoft.Extensions.Configuration.Abstractions" Version="3.1.16" />
```

Comment:
Nothing major, but could you please change its version to 2.1.1 so that our case covers as broad as possible?

Reply:
Done, thank you!

Reply:
Yeah, I agree that loading all the transitive dependencies is better, there can be a scenario where classes with HTTP triggered functions can be unloadable, in that case, `GetLoadableTypes()` doesn't work. However, I haven't seen such issues yet, so I would suggest applying my fix and enhance it in the future if somebody needs it.